reading/phoenix_authentication.livemd

```diff
@@ -1004,10 +1012,8 @@ end
 
 To fix these tests, we need to ensure the created entry belongs to a user, and that the user is logged in.
 
-The `Diary.AccountsFixture` provides a `user_fixture/1` function for creating users, and a `log_in_user/2` function for logging in users.
-
-Import the `Diary.AccountsFixture` to get access to these functions in our tests.
-It's conventional for imports to be grouped together, so put the import with out other `Diary.EntriesFixture`.
+The `Diary.AccountsFixture` module provides a `user_fixture/1` function for creating users. Import the module to get access to this function in our tests.
```

Comment:
In testing, the `log_in_user/2` function is defined in `test/support/conn_case.ex`, not in the AccountsFixture. At least, that's how it appears to me.

Reply:
I believe you are correct!